### DISCUSSION TITLE
Coming soon...

### DISCUSSION THREAD

**Discussion by bailey** [community/community#7872] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone! 👋🏼 We are so excited to share our new feature, **Lists** with all of you. ⭐ Lists will begin rolling out in phases throughout the next month. 

We'll be back with more info for you soon. Once you've gained access, come back [here](https://github.com/github/feedback/discussions/categories/lists-feedback) and tell us what you think. 👀

**Comment by johngagefaulkner** [comment]
From someone approaching 1,000 starred repos, with no end in sight, Lists has been a wonderfully useful feature. Eventually I’d like to be able to choose a default sorting for all my lists (my preference is alphabetical) rather than its current functionality of sorting by most recently updated. Thanks again for the feature!

**Comment by Bob4F** [comment]
Hi I've been on the whitelist waiting list, can I get a whitelist please?

**Comment by adityamenon** [comment]
Love this feature!

**Comment by inickt** [comment]
Looks like I have the lists feature on mobile but not in my browser– is it intentional to only be "half" in the feature preview? #8291 does show a web UI. Super excited for this to roll out further!

**Comment by DrSensor** [comment]
I've starred 3.7k repo 😂 and recently noticed this new feature and thinking to do curation on my spare time. Unfortunately the `?tab=stars` doesn't look good on mobile screen since long ago 😞

**Comment by michellemerrill** [comment, marked as answer]
Please see our latest announcement for Lists here: https://github.com/github/feedback/discussions/8618#discussion-3732895
